e2e/scripts/pre-dump.sql

```diff
@@ -0,0 +1,147 @@
+\set ECHO all
+\set ON_ERROR_STOP 1
+
+CREATE EXTENSION promscale;
+
+SELECT _prom_catalog.set_default_value('ha_lease_timeout'::text, '200 hours'::text);
+
+SELECT _prom_catalog.get_or_create_metric_table_name('cpu_usage');
```

Comment:
I'd like to see a metric with a MixedCaseNameThatIsLongerThan63Chars as well in these tests

Reply:
I'd also like a test with continuous aggs and  our cagg views

Reply:
made issues for these